tests/rpc/test_rpc_apiserver.py

```diff
@@ -406,6 +406,7 @@ def test_api_cleanup(default_conf, mocker, caplog):
 
     apiserver = ApiServer(default_conf)
     apiserver.add_rpc_handler(RPC(get_patched_freqtradebot(mocker, default_conf)))
+    apiserver.start_api()
```

Comment:
Originally had the ApiServer and ExternalSignalController being started explicitly, but since switched back to starting in the initialization. I will remove.